plugins/events-backend/src/service/EventsPlugin.test.ts

```diff
@@ -84,13 +84,13 @@ describe('eventsPlugin', () => {
     const response1 = await request(server)
       .post('/api/events/http/fake')
```

Comment:
```suggestion
      .post('/api/events/http/fake')
      .type('application/json')
```

If you add this, it will still fail despite the stringify below as the `request.body` passed to the handler is already parsed again.

I would expect the data to be sent with that content type.

For me, this indicates that the change from `express.json()` to `express.text(...)` was not enough.

_(At first, I was wondering why we need the stringify here.)_